.github/workflows/publish.yml

```diff
@@ -0,0 +1,38 @@
+name: Publish to NPMJS and GPR
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  publish-to-npm-and-gpr:
+    runs-on: ubuntu-latest
+    steps:
+      
+      # Checkout the repo
+      - uses: actions/checkout@master
+
+      # Publish to NPMJS
+      - uses: actions/setup-node@master
+        with:
+          node-version: 12
+          registry-url: 'https://registry.npmjs.org/'
+      - name: Publish to NPMJS
+        run: |
+          npm config set //registry.npmjs.org/:_authToken=$NODE_AUTH_TOKEN
+          npm config list
+          npm publish --access public 
+        env:
+          CI: true
+          NODE_AUTH_TOKEN: ${{secrets.npm_token}}
+
+      # Publish to GitHub Package Registry
+      - uses: actions/setup-node@master
+        with:
+          node-version: 12
```

Comment:
iirc discord.js requires v16 and over nodejs version to build.

Reply:
That is only to publish the package.. also discord is in dev dependency...